deploy/webhook.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: propagator-webhook-service
+  namespace: open-cluster-management
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
```

Comment:
Do you want to use the name (i.e. `webhook-http`) here instead of the port number?

Reply:
I think `targetport` is enough